syz-manager/manager.go

```diff
@@ -540,6 +552,56 @@ func (mgr *Manager) runInstance(index int) (*Crash, error) {
 		procs = 1
 	}
 
+	var rawDlkms []byte
```

Comment:
This is a wrong place for this. This is done after booting each VM. The current code leads to bad data races. Try running manager built with -race flag.
If we collect this info, it needs to be collected once and passed back from the machine in CheckArgs defined in pkg/rpctype/rpctype.go.